skops/_min_dependencies.py

```diff
@@ -13,6 +13,7 @@
     "scikit-learn": ("0.24", "install", None),
     "huggingface_hub": ("0.8.1", "install", None),
     "modelcards": ("0.1.6", "install", None),
+    "tabulate": ("0.8.10", "install", None),
```

Comment:
Do we need the latest release? Can we not do with `0.8.7` or `0.8.8`?

Reply:
Just as a data point, in skorch we use `>=0.7.7` and never had any trouble. `0.8.8` looks reasonable, it's been out for 2 years and added
> Python 3.9 support, 3.10 ready

Reply:
changed it to 0.8.8 @BenjaminBossan @adrinjalali